src/aavrutiDJun20/BankingSystem.java

```diff
@@ -0,0 +1,86 @@
+package aavrutiDJun20;
+
+public class BankingSystem {
+
+	int user2printBalCount = 0;
+	int currentBalance = 0;
+	int debitCount = 0;
+	int creditCount = 0;
+	int printBalanceCount = 0;
```

Comment:
Remove the initialization of all instance variables.

Reply:
Done